cmd/mdmctl/config.go

```diff
@@ -182,20 +189,30 @@ func migrateCmd(args []string) error {
 }
 
 func validateServerURL(serverURL string) (string, error) {
-	if serverURL != "" {
-		if !(strings.HasPrefix(serverURL, "http") ||
-			strings.HasPrefix(serverURL, "https")) {
-			serverURL = "https://" + serverURL
-		}
-		u, err := url.Parse(serverURL)
-		if err != nil {
-			return "", err
-		}
-		u.Path = "/"
-		serverURL = u.String()
+	if serverURL == "" {
+		return serverURL, errors.New("bad input: server-url must be provided.")
+	}
+
+	if !(strings.HasPrefix(serverURL, "http") ||
+		strings.HasPrefix(serverURL, "https")) {
+		serverURL = "https://" + serverURL
 	}
+	u, err := url.Parse(serverURL)
+	if err != nil {
+		return "", err
+	}
+	u.Path = "/"
+	serverURL = u.String()
+
 	return serverURL, nil
+}
+
+func validateAPIToken(apiToken string) (string, error) {
```

Comment:
IMO this function is unnecessary, just check the string directly before assigning.

Reply:
Updated in fixup bfbe930